Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -150,47 +156,40 @@ void InterpreterHandler(fextl::string* Filename, const fextl::string& RootFS, fe
   // Is the file large enough for shebang
   if (ReadSize <= 2) {
     close(FD);
-    return;
+    return true;
```

Comment:
Why does this return success?

Reply:
Function is returning if the file exists. Doesn't really matter, trying to execute a malformed file that is 2 bytes or less will error out anyway. Switched it to false.